components/alert/alert-toast.js

```diff
@@ -118,11 +141,45 @@ class AlertToast extends LitElement {
 		}
 	}
 
+	_closeTimerStart() {
+		clearTimeout(this._setTimeoutId);
+		if (!this.noAutoClose && !this._hasFocus && !this._hasMouse) {
```

Comment:
I think you want the focus and mouse checks to move inside the timeout. Otherwise I could mouse over, mouse out, then focus and it'd close.

Reply:
Hmm actually no it wouldn't, since focusing would cancel the timeout. So probably OK!